zpool-features(5) and other man page fixes

rlaager opened the pull request:
### Motivation and Context
This fixes some issues with zpool-features(5) and zfs-module-parameters(5)

### Description
- `multi_vdev_crash_dump`'s information now lines up visually.
- `encryption depends on `bookmarks_v2`
- `bookmark_v2` depends on `bookmarks`
- Document `send_holes_without_birth_time` in zfs-module-parameters(5). This is a dependency of the following change.
- Document that `hole_birth` is effectively useless to fix #8642 (which I opened).
- Change wording in zfs-module-parameters.5 (from a review comment here by @tcaputi)
- Correct GUID for `large_blocks`
- Add missing formatting for `sha512`
- Standardize `.RE` placement
- Alphabetize zpool-features.5 by short name
- Eliminate useless double bolding commands in zpool-features.5 and and zstreamdump.8 (the only two places where this occurred).

### How Has This Been Tested?
I looked at the man pages with `man`.

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:
- [x] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [x] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
- [x] All new and existing tests passed.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

rlaager commented:
It seems a little weird that we have bookmarks (plural) and bookmark_v2 (singular bookmark). I guess this was done in the style of spacemap_v2.

Are we happy with zpool-features.5 being sorted (roughly) by order of feature add, as opposed to alphabetically by feature (short) name?

rlaager commented:
Also, should DEPENDENCIES be recursed out. That is, should encryption say something like this:
```
DEPENDENCIES           bookmarks, bookmark_v2, extensible_dataset
or
DEPENDENCIES           bookmark_v2 (bookmarks), extensible_dataset
````

tcaputi commented on man/man5/zpool-features.5:
```diff
@@ -383,6 +383,13 @@ READ\-ONLY COMPATIBLE	no
+This feature has/had bugs, the result of which is that, if you do a
+\fBzfs send -i\fR (or \fB-R\fR, since it uses \fB-i\fR) from an affected
+dataset, the receiver will not see any checksum or other errors, but will not
+match the source.  Its use by \fBzfs send -i\fR has been disabled by default.
```
but the resulting destination snapshot will not match....

rlaager replied:
Fixed, in this PR and the wiki.

tcaputi commented on man/man5/zfs-module-parameters.5:
```diff
@@ -374,6 +370,19 @@ angular velocity disk drive.
+.ad
+.RS 12n
+When set, the hole_birth optimization will not be used, and all holes will
+always be sent on zfs send. Useful if you suspect your datasets are affected
```
This is useful if....

rlaager replied:
This was not my text. My change only moved this existing in the file. I have applied your change as part of this PR.

rlaager commented:
> > Are we happy with zpool-features.5 being sorted (roughly) by order of feature add, as opposed to alphabetically by feature (short) name?
> 
> I think it would be best to put these all in alphabetical order by short name. Since the order added differs between platforms. I've wanted to do this for a while now.

Done, in a separate commit consisting _only_ of that change, which I verified as follows:
diff -u <(grep .RE -B1 zpool-features.5 | sort) <(grep .RE -B1 zpool-features.5.new | sort)

This required a couple more dependent commits to get to that point.

> > Also, should DEPENDENCIES be recursed out.
> 
> My feeling is only listing the immediate dependency is more clear. So I'd suggest leaving this as-is.

Sounds good.

> Looks good, just a few comments.

Were there meant to be more comments?

rlaager commented:
I didn't mention these before: I fixed a typo I introduced of bookmarks_v2. I added some missing formatting on the sha512 section (though it didn't seem to matter, which makes me wonder if more can be removed through the whole file). I fixed `org.open-zfs:large_block` to be `org.open-zfs:large_blocks` (plural), which I verified is what the actual code uses.

I also pushed a fix to eliminate useless double bolding.
